tests/test_adaption.py

```diff
@@ -130,7 +130,7 @@ def __len__(self):
 class TestAdaption(unittest.TestCase):
     @classmethod
     def setUpClass(cls):
-        interface.is_development = True
+        pure_interface.set_is_development(True)

```

Comment:
Is there any threshold for setting up a github action to automatically run the tests in this repo? Or does it just require someone having the time and wanting to?

Reply:
I think it just requires having time and wanting to.